src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -0,0 +1,652 @@
+# This file collects all methods that have been updated to work with parabolic systems of equations
+#
+# assumptions: parabolic terms are of the form div(f(u, grad(u))) and
+# will be discretized first order form as follows:
+#               1. compute grad(u)
+#               2. compute f(u, grad(u))
+#               3. compute div(u)
+# boundary conditions will be applied to both grad(u) and div(u).
+function rhs_parabolic!(du, u, t, mesh::TreeMesh{2}, equations_parabolic::AbstractEquationsParabolic,
+                        initial_condition, boundary_conditions, source_terms,
+                        dg::DG, dg_parabolic, cache, cache_parabolic)
+
+  # Reset du
+  @trixi_timeit timer() "reset ∂u/∂t" reset_du!(du, dg, cache)
+
+
+  @unpack u_transformed, u_grad, viscous_flux = cache_parabolic
```

Comment:
`gradients` sounds good to me.

Reply:
How about `gradients_parabolic`? `gradients` is good too, but I think `gradients_parabolic` makes the name more searchable.

Reply:
The existence of `gradients_parabolic` implies to me that there is also a `gradients_hyperbolic` (or something similar). Although I do understand you argument about searchability 😬

Reply:
`gradients_u` would also work. I leaned towards `gradients` because `gradients_u` might imply they are gradients of the solution, rather than of the transformed solution (e.g., `cons2prim`).

Reply:
👍 to `gradients`

Reply:
Done in 020fb1504f9ad1af69418425dd2edc559406622d